core.cc

```diff
@@ -90,10 +93,12 @@ NAN_METHOD(SysLog) {
 
     if (message) {
         Nan::AsyncQueueWorker(new Worker(callback, priority, message));
-    } else if(!callback->IsEmpty()) {
```

Comment:
Ah, I see now.  LGTM but you can replace the `info[2]->IsFunction()` check below with a `callback != NULL`.